samples/42.scaleout/src/main/java/com/microsoft/bot/sample/scaleout/BlobStore.java

```diff
@@ -32,7 +32,17 @@
     private final CloudBlobContainer container;
     private final ObjectMapper objectMapper = new ObjectMapper().findAndRegisterModules();
 
-    public BlobStore(String accountName, String accountKey, String containerName) throws URISyntaxException, StorageException {
+    /**
+     * The constructor of the {@link BlobStore} class.
+     * @param accountName The account name of the Storage Account.
+     * @param accountKey The account key of the Storage Account.
+     * @param containerName The container name.
+     * @throws URISyntaxException Throws an {@link URISyntaxException}
+     *                            if the {@link CloudStorageAccount} cannot be created.
+     * @throws StorageException Throws a {@link StorageException} if the specified container doesn't exist.
+     */
+    public BlobStore(String accountName, String accountKey, String containerName)
+        throws URISyntaxException, StorageException {
```

Comment:
Exactly this should be handled here instead of throwing exceptions, what do you think?

Reply:
I'm agree with you, I'll apply it